ts/att/timeSpan.ts

```diff
@@ -3,19 +3,32 @@ export class TimeSpan {
     public totalSeconds: number;
 
     public constructor(hours: number, minutes: number, seconds: number) {
-        if (hours < 0 || minutes < 0 || seconds < 0) {throw RangeError; }
+        if (hours < 0 || minutes < 0 || seconds < 0) {
+            throw RangeError;
+        }
         this.totalSeconds = seconds + (hours * 60 + minutes) * 60;
     }
 
     public hours(): number {
-        return this.totalSeconds / 3600;
+        const numberOfSecondsPerHour = 3600;
+        return this.totalSeconds / numberOfSecondsPerHour;
     }
 
     public minutes(): number {
```

Comment:
I dont understand this. Why `this.totalSeconds % numberOfSecondsPerHour / numberOfSecondsPerMinute` ?